image-segmentation/steps/data_steps.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+from torch.utils.data import DataLoader
+from zenml.logger import get_logger
+from zenml.steps import Output, step
+
+from .core_src.configs import PreTrainingConfigs
+from .core_src.data.data_loader import CustomDataLoader
+from .core_src.data.data_process import ProcessData
+from .core_src.data.prepare_data import PrepareDataFrame
+
+
+logger = get_logger(__name__)
+
+
+@step
+def prepare_df() -> Output(processed_dataframe=pd.DataFrame):
+    """
+    It processes and manipulates the masked df.
+    """
+    try:
+        prep_df = PrepareDataFrame("./data/archive/updated_files.csv")
```

Comment:
Hi there @ayush714 , thank you for putting out this PR. I was trying to run the pipeline but I could not find this file here. Do you recall how to obtain this `.csv` file?